contents/handbook/comms/customer-support.md

```diff
@@ -81,6 +81,8 @@ When support engineers need to escalate issues to other engineering teams for de
 - Normal priority escalated ticket response target: 4 business days
 - Low priority escalated ticket response target: N/A
 
+> **_NOTE:_** The targets are for a reply to the user. If the escalation turns out to be a bug or feature request, it doesn't have to be solved by the response target date, we just need to reply to the user.  That reply may be to let them know it won't be fixed right away, but that we have opened a bug report or feature request.
```

Comment:
Probably worth noting here whether we expect tickets to be solved, pending, on-hold, etc, in such situations.

Reply:
Ah yes, good call! Added and punched the re-request review button.